README.md

```diff
@@ -1,70 +1,103 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Currency
 
-## Available Scripts
+> Microverse's React capstone project
 
-In the project directory, you can run:
+![screenshot](./screenshot-1.png)
+![screenshot](./screenshot-2.png)
 
-### `npm start`
+A mobile web single-page application to check the daily exchange rate of the world's currencies. It makes use of React, Redux, and a currency API.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Built With
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+- HTML & CSS
+- JavaScript
+- React
+- Redux
+- API
 
-### `npm test`
+## Live Demo
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Check out a live demo of this project by [clicking here.](https://currency-x.netlify.app/)
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Getting Started
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+To get a local copy up and running follow these steps.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-### `npm run eject`
+### Prerequisites
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Setup
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+1. Download or clone the repository's contents.
+2. Go to the project's root directory.
+```
+cd currency
+```
+3. Install all the dependencies:
+```
+npm install
```

Comment:
- [ ] Kindly include instructions to run tests.

Reply:
Done.